src/main/java/org/example/Main.java

```diff
@@ -1,7 +1,120 @@
 package org.example;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+
 public class Main {
+
+    static List<String> options = new ArrayList<>(List.of("add", "remove", "list", "exit"));
+    static List<List<String>> tasks = new ArrayList<>();
+    static Scanner scanner = new Scanner(System.in);
+
     public static void main(String[] args) {
-        System.out.println("Hello, World!");
+        readTasks("tasks.csv");
+        selectOption();
+    }
+
+    public static void readTasks(String fileName) {
+        Path path = Paths.get(fileName);
+        try {
+            List<String> lines = Files.readAllLines(path);
+            for (String line : lines) {
+                String[] words = line.split(",");
```

Comment:
CLEAN-CODE: vice popisne me prijde nazvat polozky ukolu jako items

```suggestion
                final String[] items= line.split(",");
```